src/pages/home/sidebar/OptionRow.js

```diff
@@ -140,9 +140,13 @@ const OptionRow = ({
                                         avatarImageURLs={option.icons}
                                         size={mode === 'compact' ? 'small' : 'default'}
                                         secondAvatarStyle={[
-                                            getSecondAvatarStyle(backgroundColor),
-                                            optionIsFocused && getSecondAvatarStyle(focusedBackgroundColor),
-                                            hovered && !optionIsFocused && getSecondAvatarStyle(hoveredBackgroundColor),
+                                            getBackgroundAndBorderStyle(backgroundColor),
+                                            optionIsFocused
```

Comment:
Why the ternary instead of `optionIsFocused && getBackgroundAndBorderStyle(focusedBackgroundColor),` ?

Reply:
Mostly to silence propTypes warnings. We're validating for an object and then sending a boolean.

Reply:
Cool, that's a good reason!